app/views/core/CreateAccountModal/BasicInfoView.coco.coffee

```diff
@@ -298,10 +298,6 @@ module.exports = class BasicInfoView extends CocoView
           window.tracker?.trackEvent 'Facebook Login', category: "Signup", label: 'Facebook'
         )
 
-      trackerCalls.push(
-        globalVar.application.tracker?.trackEvent 'Finished Signup', category: "Signup", label: loginMethod
```

Comment:
Removing this is going to get rid of all "Finished Signup" events in all our analytics destinations, right?

Reply:
so it was actually being tracked twice. A few lines above in this file, you will see `me.signupWithPassword` which already tracks this event: https://github.com/codecombat/codecombat/blob/master/app/models/User.coffee#L555

We do the same on google and Facebook.

I need to check Clever one though to see whether we check that!!